packages/gatsby-cli/README.md

```diff
@@ -45,11 +45,40 @@ to find out how you can set up an HTTPS development server using Gatsby.
 At the root of a Gatsby site run `gatsby build` to do a production build of a
 site.
 
+#### Options
+
+```
```

Comment:
@KyleAMathews I used the indented blocks because that was there, but what about a table, e.g.

|Command|Description|
|:-----:|-----------|
|`-H` `--host`|Set host. Defaults to localhost|
|`-p` `--port`|Set port. Defaults to 8000|
|`-o` `--open`|Open the site in your browser for you|
|`--prefix-paths`|Serve site with link paths prefixed (set prefix in your config).|

Reply:
👍